### PR TITLE
Jk/release 18.5.1

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-3920](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3920)

## Changes

Version up commits for 18.5.1 patch release. 

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by ckhowardASDC** [comment]
Can you also include the error-handling fixes related to CUMULUS-3902? https://bugs.earthdata.nasa.gov/browse/CUMULUS-3902

ASDC will need the pdr-status-check task to be fixed for our PAN/PDR workflow.

File: scripts/generate_records/package.json [hunk 1]
```diff
@@ -41,4 +41,4 @@
     "@cumulus/api-client": "18.5.0",
```
Comment (etcart) [inline scripts/generate_records/package.json:41]:
```suggestion
    "@cumulus/api-client": "18.5.1",
```

File: tasks/update-granules-cmr-metadata-file-links/package.json [hunk 2]
```diff
@@ -40,16 +40,16 @@
...
+    "@cumulus/common": "18.5.1",
     "@cumulus/cumulus-message-adapter-js": "2.2.0",
-    "@cumulus/distribution-utils": "18.5.0",
-    "@cumulus/logger": "18.5.0",
+    "@cumulus/distribution-utils": "18.5.1",
+    "@cumulus/logger": "18.5.1",
     "lodash": "^4.17.15"
   },
   "devDependencies": {
     "@cumulus/aws-client": "18.4.0",
```
Comment (etcart) [inline tasks/update-granules-cmr-metadata-file-links/package.json:52]:
```suggestion
    "@cumulus/aws-client": "18.5.1",
```

**Review by etcart — CHANGES_REQUESTED** [review]
a few changes requested, also:
I believe website/package.json>version should be 18.5.1 not 18.5.0
and I believe now that you are building 18.5.1 and releasing it, this is the moment where you can remove the jsonpath-plus allowlist

File: scripts/generate_records/package.json (hunk 1 above)
Comment (Jkovarik) [reply scripts/generate_records/package.json:41]: I'm confused by this suggestion - api client *is* being updated to 18.5.1 already in the PR

**Comment by Jkovarik** [comment]
@ckhowardASDC  Hey Chris - 

Happy to respond here, but those sorts of requests should absolutely go through I&A/PO/SM channels.      Happily, we *did* include it - this PR is just the version up/paperwork, the backport candidate PR was here: https://github.com/nasa/cumulus/pull/3833 *and did include 3902*!  

Check out the CHANGELOG in this branch for the full set of candidates.

**Comment by Jkovarik** [comment]
@etcart  *technically* the audit should fail until the release is done, but given we skip audit in this release process..... fair enough.

File: tasks/update-granules-cmr-metadata-file-links/package.json (hunk 2 above)
Comment (Jkovarik) [reply tasks/update-granules-cmr-metadata-file-links/package.json]: 3be4c3e2ef2

File: scripts/generate_records/package.json (hunk 1 above)
Comment (Jkovarik) [reply scripts/generate_records/package.json:41]: Talked about this in a hangout, going to remove the extra line from the dev dependencies.

Comment (Jkovarik) [reply scripts/generate_records/package.json]: afde3d0e3cd

**Review by etcart — APPROVED** [review]

**Comment by Jkovarik** [comment]
@etcart  pushed up the CI fixes from main we're talking about in chat.   Will merge once CI runs.

**Pull request merged**